conformance/system_macros/set_macros.ion

```diff
@@ -0,0 +1,10 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Test Cases:
+// set_macros can be invoked by name, address, qualified address, qualified name, binary address, and binary system e-expression op code.
+// the argument values may be zero or more macro definitions
+//    TODO: Could it also be a module or an export?
```

Comment:
Off-hand I think yes for both this and `add_macros`? `mod_a::*` and `macro_foo` both seem reasonable.

Though there is something to be said for the charm of macro-shaped parameters here:
```ion
(:set_macros
   (foo () ...)
   (bar () ...)
   (baz () ...)
)
```
Probably not worth precluding the other options, though. I suppose we could have a separate `def_macros` that only took macro definitions.